garageScript/c0d3-app
# feat(DOJO): Create main mentor page

Related to #2029 

### Changes

- Create the main mentor page under `curriculum/[lessonSlug]/mentor` which is a *literal* modified copy of the [exercises page](https://c0d3.com/exercises/js1) including its code and test file.
- Add another path to the `NavBar` in the exercises page that link to `/curriculum/[lessonSlug]/mentor`
- Move the page for adding an exercise to `/mentor/addExercise`

<details>
<summary>
Page Image
</summary>
<img width="1195" alt="image" src="https://user-images.githubusercontent.com/35906419/193118777-7ab197d0-3cf8-4608-8e18-87b157a24369.png">

</details>


### Subsequent changes

1. Create a query to get all the mentor's created exercises based on the user ID
2. Modify the exercise card to not show ANSWERED or NOT ANSWERED states
3. Add a back button to the `addExercise` page

### Testing

1. Go to `/curriculum/js0/mentor`
2. Confirm the pages show up correctly
3. Click `MENTOR EXERCISES` in the `NavBar`
4. Confirm the `addExercise` page show up correctly
